go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cryostatio/cryostat-operator
 
-go 1.15
+go 1.16
```

Comment:
Does this mean the project will use `go` v1.16 in its dependencies while the `Dockerfile` will use v1.17 to build the image?

Reply:
This specifies that you must be using at least Go 1.16 to build the operator. https://go.dev/doc/modules/gomod-ref#go
If you have Go 1.16 installed locally, then you'll still be able to build the operator locally with `make manager`. We can look at upgrading this to 1.17+ once available on more platforms. For example, Fedora 35 is still using 1.16.
When building the container image (e.g. `make oci-build`), it will build using 1.17.

Reply:
Makes sense, thanks!